README.md

```diff
@@ -2,74 +2,83 @@
 
 ## Description
 
-This tool is a first draft of connecting an LMS interface to a Graylog server to create a dataset 
-of learning logs.
+This tool is dedicated to generate learning logs from Edx platform to feed dashboard development testing datasets.
+It is intended to be used locally.
 
-It is intended to be used locally. 
-Currently, it is using [OpenEdx](https://github.com/openfun/openedx-docker)'s LMS service 
-(with the fun-flavored dogwood's release).
+It has been duplicated from [OpenEdx](https://github.com/openfun/openedx-docker) fun-flavored dogwood's release
+and integrates a Graylog stream server connected to the LMS service.
 
-## Getting started
+## Usage
 
-- Clone the current repository.
+### Launch Edx platform
 
-- When using the tool for the first time, you need to run migrations. Run the following command:
+1. Run the following command to launch the Edx platform:
 
 ```
-make migrate
+make bootstrap
 ```
 
-The tool is now ready for use ! :rocket:
+> Note: as the command comprises migration, it can be long.
+> Don't be afraid to wait several minutes.
 
-## Usage
+2. When connections are established, you can access:
 
-### Run the learning analytics playground tool
+   - the LMS server on [localhost:8072](http://localhost:8072/login)
+   - the CMS server on [localhost:8082](http://localhost:8072/signin)
+   - the Graylog server on [localhost:9000](http://localhost:9000)
 
-The whole learning tool (both LMS and graylog servers) are launched in one command. 
+3. Two users are available for testing the LMS/CMS:
 
-1. Run the following command:
+   - admin: `admin@example.com`:`admin`
+   - student: `edx@example.com`:`edx`
 
-```
-make run
-```
+Though it is possible to create customized users at your convenience.
 
-2. When connections are established, you can access:
+### Run end-to-end tests
 
-    * the LMS server on [localhost:8072/login](http://localhost:8072/login)
-    * the CMS server on [localhost:8082/signin](http://localhost:8072/signin)
-    * the Graylog server on [localhost:9000](http://localhost:9000)
+End-to-end tests are used in the playground to generate factice learning logs.
 
+As this project is evolving, the tests will progressively comprises a wide panel of use cases.
+For now, the tests allow to generate logs about:
 
-3. Two users are available for testing the LMS/CMS:
+- the registration of a student to the platform (LMS)
+- the creation of a course (CMS)
+- the registration of a student to a course (LMS)
```

Comment:
Not sure we need to mention this in the project's README as I'm pretty sure we will not maintain this list. WDYT?

Reply:
You're right, I thought of moving the current README in a documentation when the project will be bigger. Hence, such infos could be necessary.

Reply:
I would remove this for now, maintain a changelog, and eventually write a real documentation but I think it's too early for this.

Reply:
Let's do it like that